src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -279,6 +279,11 @@ Status CoreWorkerMemoryStore::Get(const std::vector<ObjectID> &object_ids,
                  /*abort_if_any_object_is_exception=*/true);
 }
 
+bool CoreWorkerMemoryStore::ObjectHasOwner(const ObjectID &object_id) {
+  rpc::Address unused_owner_address;
+  return ref_counter_->GetOwner(object_id, &unused_owner_address);
```

Comment:
@jjyao - the CPP tests seem to fail when the `ref_counter_` is not properly initialized (specifically, its `mutex_` and `object_id_refs_` objects seem unallocated for some reason... do you know what I may have missed at initialization?

Reply:
(I've figured this out - lmk if the fix looks okay)